hack/builder/Dockerfile

```diff
@@ -26,7 +26,6 @@ RUN dnf install -y dnf-plugins-core && \
         libstdc++-static \
         glibc-devel \
         findutils \
-        gradle \
```

Comment:
We need to keep this. https://travis-ci.org/github/kubevirt/kubevirt/builds/720719969#L428

Reply:
That package was removed from fc31. Had to add a manual install to reenable it.